Features2/tests/features/searchContact.feature

```diff
@@ -0,0 +1,28 @@
+Feature:Search contact
+    As a user
+    I want to search the contact from the contact list
+    So I can easily get the required contact from the list
+    
+    Background:
+     Given the user has added following contacts
+        | name | Email | Phone | Contact Type |
+        | abcde | abcde@gmail.com | 00000000 | Professional |
+        | xyz | xyz@gmail.com | 111111111 | Personal |
+        | example |example@gmail.com | 22222222222| Professional |
+        | example2 | example2@gmail.com |3333333333|Personal|
+        | abcd1 |sabhdb@gmail.com | 9876666 | Personal |
+        |  cvbn |cvbn@gmail.com | 090099090 | Professional |
+        | mnbv | mnvb@gmail.com | 098876677 | Professional |
+
+     Scenario:Search contact by email
+        When the user enters the letter of email in the search icon
```

Comment:
i think you have to pass some value to search contact and get the required result right? so pass the value.